src/main/java/hlf/java/rest/client/service/impl/ChannelServiceImpl.java

```diff
@@ -116,6 +129,14 @@ public ClientResponseModel joinChannel(ChannelOperationRequest channelOperationR
     }
   }
 
+  @Override
+  public Set<String> getChannelMembersMSPID(String channelName) {
```

Comment:
Can we check and add test cases for a couple of scenarios -
1) Channel does not exists
2) Channel not being passed or passed as ""(empty string) while calling

Reply:
1. If channel doesn't exist then inside this method `gateway.getNetwork(channelName)` a new channel is getting.
2.  Added a check for handling empty string scenario, for channel not being passed that is already handled at controller level.

Reply:
Can we add test case for case 1. Directly call getChannelMembersMSPIDTest, let's test the response.

Reply:
testcase added